### ISSUE TITLE
iOS build crash

### ISSUE THREAD

**Issue by rtviwe** [Innim/flutter_login_vk#2]
```    2 warnings generated.
    error: the following command failed with exit code 0 but produced no further output
    CompileC /Users/rtviwe/Library/Developer/Xcode/DerivedData/Runner-gzvhtvargsftfrctgwkeycfoyigt/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/AppAuth.build/Objects-normal/x86_64/OIDExternalUserAgentIOS.o /Users/rtviwe/FlutterProjects/GoArctic/frontend/go_arctic/ios/Pods/AppAuth/Source/AppAuth/iOS/OIDExternalUserAgentIOS.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
    /Users/rtviwe/.pub-cache/hosted/pub.dartlang.org/flutter_login_vk-0.1.1/ios/Classes/SwiftFlutterLoginVkPlugin.swift:163:38: error: cannot use optional chaining on non-optional value of type 'Bool'
                    "online": user.online?.toBool(),
                              ~~~~~~~~~~~^
    /Users/rtviwe/.pub-cache/hosted/pub.dartlang.org/flutter_login_vk-0.1.1/ios/Classes/SwiftFlutterLoginVkPlugin.swift:163:40: error: value of type 'Bool' has no member 'toBool'
                    "online": user.online?.toBool(),
                              ~~~~~~~~~~~~ ^~~~~~
    /Users/rtviwe/.pub-cache/hosted/pub.dartlang.org/flutter_login_vk-0.1.1/ios/Classes/SwiftFlutterLoginVkPlugin.swift:164:51: error: cannot use optional chaining on non-optional value of type 'Bool'
                    "onlineMobile": user.online_mobile?.toBool(),
                                    ~~~~~~~~~~~~~~~~~~^
    /Users/rtviwe/.pub-cache/hosted/pub.dartlang.org/flutter_login_vk-0.1.1/ios/Classes/SwiftFlutterLoginVkPlugin.swift:164:53: error: value of type 'Bool' has no member 'toBool'
                    "onlineMobile": user.online_mobile?.toBool(),
                                    ~~~~~~~~~~~~~~~~~~~ ^~~~~~
    /Users/rtviwe/FlutterProjects/GoArctic/frontend/go_arctic/ios/Pods/GTMAppAuth/Source/GTMAppAuthFetcherAuthorization+Keychain.m:31:26: warning: 'unarchiveObjectWithData:' is deprecated: first deprecated in iOS 12.0 - Use +unarchivedObjectOfClass:fromData:error: instead [-Wdeprecated-declarations]
          [NSKeyedUnarchiver unarchiveObjectWithData:passwordData];
                             ^
    In module 'Foundation' imported from /Users/rtviwe/FlutterProjects/GoArctic/frontend/go_arctic/ios/Pods/GTMAppAuth/Source/GTMKeychain.h:19:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSKeyedArchiver.h:152:1: note: 'unarchiveObjectWithData:' has been explicitly marked deprecated here
    + (nullable id)unarchiveObjectWithData:(NSData *)data API_DEPRECATED("Use +unarchivedObjectOfClass:fromData:error: instead", macosx(10.2,10.14), ios(2.0,12.0), watchos(2.0,5.0), tvos(9.0,12.0));
    ^
    /Users/rtviwe/FlutterProjects/GoArctic/frontend/go_arctic/ios/Pods/GTMAppAuth/Source/GTMAppAuthFetcherAuthorization+Keychain.m:41:48: warning: 'archivedDataWithRootObject:' is deprecated: first deprecated in iOS 12.0 - Use +archivedDataWithRootObject:requiringSecureCoding:error: instead [-Wdeprecated-declarations]
      NSData *authorizationData = [NSKeyedArchiver archivedDataWithRootObject:auth];
                                                   ^
    In module 'Foundation' imported from /Users/rtviwe/FlutterProjects/GoArctic/frontend/go_arctic/ios/Pods/GTMAppAuth/Source/GTMKeychain.h:19:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.1.sdk/System/Library/Frameworks/Foundation.framework/Headers/NSKeyedArchiver.h:47:1: note: 'archivedDataWithRootObject:' has been explicitly marked deprecated here
    + (NSData *)archivedDataWithRootObject:(id)rootObject API_DEPRECATED("Use +archivedDataWithRootObject:requiringSecureCoding:error: instead", macosx(10.2,10.14), ios(2.0,12.0), watchos(2.0,5.0), tvos(9.0,12.0));
    ^
    2 warnings generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description
Could not build the application for the simulator.
Error launching application on iPhone 11 Pro Max.
```

**Comment by rtviwe** [comment]
`flutter doctor -v`
```
[✓] Flutter (Channel beta, 1.23.0-18.1.pre, on Mac OS X 10.15.7 19H2 x86_64,
    locale ru-RU)
    • Flutter version 1.23.0-18.1.pre at /Users/rtviwe/flutter
    • Framework revision 198df796aa (4 weeks ago), 2020-10-15 12:04:33 -0700
    • Engine revision 1d12d82d9c
    • Dart version 2.11.0 (build 2.11.0-213.1.beta)

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
    • Android SDK at /Users/rtviwe/Library/Android/sdk
    • Platform android-30, build-tools 29.0.3
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.1, Build version 12A7403
    • CocoaPods version 1.10.0

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.51.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.16.0

[✓] VS Code (version 1.48.2)
    • VS Code at /Users/rtviwe/Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.16.0

[✓] Connected device (4 available)
    • GM1910 (mobile)            • 79db564f                             •
      android-arm64  • Android 10 (API 29)
    • iPhone 11 Pro Max (mobile) • 05CC006F-978C-421D-B7FD-CD4D1FF2BD22 • ios
      • com.apple.CoreSimulator.SimRuntime.iOS-14-1 (simulator)
    • Web Server (web)           • web-server                           •
      web-javascript • Flutter Tools
    • Chrome (web)               • chrome                               •
      web-javascript • Google Chrome 86.0.4240.183

• No issues found!
```

**Comment by greymag** [comment]
Hey! Thanks for the report. Fixed in version 0.2.0, already on [pub.dev](https://pub.dev/packages/flutter_login_vk).
